prefect/helm/prefect/values.yaml

```diff
@@ -27,6 +27,12 @@ prefect-orion:
       postgresql: 5432
     externalHostname: plural-postgres-prefect
 
+test-base:
+  enabled: false
+  repository: prefect
+  testName: prefect-integration
+  promoteTag: warm
+
```

Comment:
You'll need to specify the steps with the latest version of the test-base helm chart. See https://github.com/pluralsh/module-library/blob/main/helm/test-base/values.yaml.

Reply:
we should modify the latest version so these are the default.  I avoided using it because i didn't want to have to specify them

Reply:
and more importantly, there's a chance prefect will own the packaging here, so a smoother experience is better.

Reply:
We can try and streamline the upstream test chart a bit more. But they’ll need to have the newer chart to be able to run useful tests.